components/skills/SkillCard.tsx

```diff
@@ -1,15 +1,26 @@
+'use client';
+
 import Image from 'next/image';
 
 import { SkillsType } from '@/types';
+import { fadeIn } from '@/utils/motion';
+import { motion } from 'framer-motion';
```

Comment:
Two things: 
- I don't believe "use client" is necessary here.
- Don't forget to list your imports by external imports followed by internal imports as listed below:

```
import Image from 'next/image';
import { motion } from 'framer-motion';

import { SkillsType } from '@/types';
import { fadeIn } from '@/utils/motion';

```